src/database/adatabasesetup.cpp

```diff
@@ -299,16 +299,18 @@ bool ADataBaseSetup::downloadTemplates()
 bool ADataBaseSetup::backupOldData()
 {
     auto database_file = aDB()->databaseFile;
-    if(database_file.exists()){
-        auto date_string = QDateTime::currentDateTime().toString(Qt::ISODate);
-        auto backup_dir = QDir(AStandardPaths::pathTo(AStandardPaths::DatabaseBackup));
-        auto backup_name = database_file.baseName() + "-backup-" + date_string + ".bak";
-        if(!backup_dir.mkpath(backup_name)){
-            DEB << "Could not create file " << backup_name << " at: " << backup_dir.path();
-            return false;
-        }
-        DEB << "Backing up old database as: " << backup_name;
+    if(!database_file.exists()) {
+        DEB << "No Database to backup, returning.";
+        return true;
+    }
+    auto date_string = QDateTime::currentDateTime().toString(Qt::ISODate);
+    auto backup_dir = QDir(AStandardPaths::pathTo(AStandardPaths::DatabaseBackup));
+    auto backup_name = database_file.baseName() + "-backup-" + date_string + ".bak";
```

Comment:
maybe keep the extension as .db since the backup is already in the file name and it makes it clearer what kind of file we are dealing with?

Reply:
actually i didnt even look at the fact that backup is there.
A suggestion for naming would be
`logbook.db -> logbook-ISODATE.db.bak`
that way have both data and backup in a clean fairly conventional way

Reply:
That's good for linux but windows is quite picky with files that have more than one `.` in them due to the whole file extension handling. Maybe `logbook.db -> logbook_bak_isodate.db`?

Reply:
If i knew this isnt a public log i would say lots of stuff... -,-
yours seems safest. Changing